### DISCUSSION TITLE
Configuring the theme in an embedded browser

### DISCUSSION THREAD

**Discussion by samuelmiver** [GMOD/jbrowse-components#3251] [Q&A] [answered]
I have successfully followed the tutorial to [embed the linear genome view](https://jbrowse.org/jb2/docs/tutorials/embed_linear_genome_view/) in a web. Now, I would like to harmonize the colors of the browser with the website that will contain it. 

Based on the page describing [how to configure the browser](https://jbrowse.org/jb2/docs/config_guide/) I know it is possible by modifying the top-level `config` file which I can replicate. However, in the embedded version the `tracks` and `assembly` files are used instead of the `config`. Where should I define the theme in this case? 

Thanks in advance.

**Comment by cmdcolin** [comment, marked as answer]
we have an example of "theming" here for the embedded component https://jbrowse.org/storybook/lgv/v2.1.5/?path=/story/theming--page

 the code sample shows that the createViewState accepts a "configuration" object with a theme

**Reply by samuelmiver** [reply]
That's great! thanks a lot @cmdcolin
